administrator/language/en-GB/plg_twofactorauth_email.ini

```
@@ -0,0 +1,26 @@
+; Joomla! Project
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_TWOFACTORAUTH_EMAIL="Two Factor Authentication - Authentication Code by Email"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_DESC="Should I automatically add the Authentication Code by Email as an option for <em>all</em> users? Useful to provide a fallback to users who lost access to their main authenticator and haven't kept a copy of the backup codes at the expense of some degree of control and security."
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_LABEL="Force Enable"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_120="Two minutes (recommended)"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_180="Three minutes"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_300="Five minutes"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_30="Half a minute"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_60="One minute"
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_DESC="A new code is generated every this many minutes. Do note that a generated code is valid for at least this much time and at most twice as much time. The higher this period is the more likely it is for the code to be brute forced, therefore the least secure your site is. A period of 2 minutes is a good trade-off between usability and security."
+PLG_TWOFACTORAUTH_EMAIL_CONFIG_TIMESTEP_LABEL="Code generation period"
+PLG_TWOFACTORAUTH_EMAIL_EMAIL_BODY="Two Factor Authentication on {sitename}"
+PLG_TWOFACTORAUTH_EMAIL_EMAIL_SUBJECT="Your {sitename} authentication code is -{code}-"
+PLG_TWOFACTORAUTH_EMAIL_ERR_INVALID_CODE="Invalid or expired code. Please reload the page to send yourself a new code. Make sure to enter the code within two minutes since you requested the code."
+PLG_TWOFACTORAUTH_EMAIL_LBL_DISPLAYEDAS="Code by Email"
+PLG_TWOFACTORAUTH_EMAIL_LBL_LABEL="Enter the code you received by email"
+PLG_TWOFACTORAUTH_EMAIL_LBL_SETUP_LABEL="Enter the code you received by email. Then click on the button below."
+PLG_TWOFACTORAUTH_EMAIL_LBL_SETUP_PLACEHOLDER="Six Digit Code"
+PLG_TWOFACTORAUTH_EMAIL_LBL_SHORTINFO="Receive 6-digit codes by email."
+PLG_TWOFACTORAUTH_EMAIL_MAIL_DESC="Sent to users from the Two Factor Authentication page when using the “Authentication Code by Email” option."
+PLG_TWOFACTORAUTH_EMAIL_MAIL_LBL="Joomla: Authentication Code by Email"
+PLG_TWOFACTORAUTH_EMAIL_XML_DESCRIPTION="<p>Use time-limited, six-digit security codes sent to you by email.</p>"
```

```suggestion
PLG_TWOFACTORAUTH_EMAIL_XML_DESCRIPTION="Use time-limited, six-digit security codes sent to you by email."
```


The html should not be needed